### DISCUSSION TITLE
Arduino and IDF Version

### DISCUSSION THREAD

**Discussion by estevaofv** [pschatzmann/ESP32-A2DP#302] [Q&A] [answered]
How can I check what is the most suitable arduino-esp32 release for compiling the latest release of this library, the same for the ESP-IDF. Thanks in advance!

**Comment by pschatzmann** [comment, marked as answer]
To find out the arduino-esp32 release, go to the -> Tools -> Board -> Board Manager and Enter ESP32 in the Filter. This will show you the installed version.
The IDF Release you find in the Release Information: the https://github.com/espressif/arduino-esp32/releases/

But honestly it should not matter too much since all releases since 1.0.6 are supported by ESP32-A2DP.
